### PR TITLE
VACMS-834: Replace all instances of [site:name] with Veterans Affairs in content metatags.

### PR DESCRIPTION
**Pull request by kevwalsh**
See https://github.com/department-of-veterans-affairs/va.gov-cms/issues/834

### CODE REVIEW

File: config/sync/metatag.metatag_defaults.node.yml [hunk 1]
```diff
@@ -7,12 +7,13 @@ _core:
 id: node
 label: Content
 tags:
-  title: '[node:title] | [site:name]'
   description: '[node:field_description:value]'
   image_src: '[node:field_media:entity:image:entity:url]'
-  og_image: '[node:field_media:entity:image:entity:url]'
```
Comment (swirtSJW) [inline config/sync/metatag.metatag_defaults.node.yml]: Seems like an unrelated loss.

Comment (kevwalsh) [reply config/sync/metatag.metatag_defaults.node.yml]: look a few lines down.
